config/pipeline.yaml

```diff
@@ -467,6 +467,22 @@ scheduler:
   - job: kbuild-gcc-10-armel
     <<: *build-k8s-all
 
+  - job: kbuild-gcc-10-armel-android-mainline
+    <<: *build-k8s-all
+    rules:
+      tree:
+      - 'android'
+      branch:
+        - 'android-mainline'
```

Comment:
Those aren't needed (already enforced by the job config), same for the ones below

Reply:
PR updated, please check